images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -87,63 +63,50 @@ RUN set -ex \
  && cp -s /opt/yarn/stable/bin/yarn /opt/yarn/stable/bin/yarnpkg /links
 
 FROM main AS final
-ARG BUILD_DIR
-ARG IMAGES_DIR
-ARG SDK_STORAGE_ENV_NAME
 ARG SDK_STORAGE_BASE_URL_VALUE
-WORKDIR /
-
-ENV ORIGINAL_PATH="$PATH"
-ENV ORYX_PATHS="/opt/oryx:/opt/yarn/stable/bin:/opt/hugo/lts"
-ENV PATH="${ORYX_PATHS}:${ORIGINAL_PATH}"
-COPY images/build/benv.sh /opt/oryx/benv
-RUN chmod +x /opt/oryx/benv
-RUN mkdir -p /usr/local/share/pip-cache/lib
-RUN chmod -R 777 /usr/local/share/pip-cache
-
-# .NET Core related environment variables
-ENV NUGET_XMLDOC_MODE=skip \
-	DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1 \
-	NUGET_PACKAGES=/var/nuget
-
-# Grant read-write permissions to the nuget folder so that dotnet restore
-# can write into it.
-RUN mkdir -p /var/nuget
-RUN chmod a+rw /var/nuget
-
-# Copy Yarn and Hugo related content
-COPY --from=nodetools-install /opt /opt
-
-# Build script generator content. Docker doesn't support variables in --from
-# so we are building an extra stage to copy binaries from correct build stage
-COPY --from=buildscriptgenerator /opt/buildscriptgen/ /opt/buildscriptgen/
-RUN ln -s /opt/buildscriptgen/GenerateBuildScript /opt/oryx/oryx
+ARG AI_KEY
 
-# Install PHP pre-reqs
-RUN ${IMAGES_DIR}/build/php/prereqs/installPrereqs.sh
-# NOTE: do not include the following lines in prereq installation script as
-# doing so is causing different version of libargon library being installed
-# causing php-composer to fail
-RUN apt-get update \
+COPY --from=pre-final /opt /opt
```

Comment:
nit: rename pre-final

Reply:
Any suggestions?

Reply:
Take a look at the new name in the latest commit that I pushed. I just named it `intermediate`

Reply:
seems like we are preparing opt folder in final in this layer .. so can it be assemble-opt-dir-for-final .... will that be too long?

Reply:
Hmm, I do not think that is necessary